pkgs/by-name/reoxide/package.nix

```
@@ -0,0 +1,85 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitea,
+  cacert,
+
+  meson,
+  ninja,
+  python3,
+  cmake,
+  pkg-config,
+
+  libllvm,
+  libclang,
+  cppzmq,
+}:
+stdenv.mkDerivation (finalAttrs: {
```

It compiles with `clangStdenv`, now:

```suggestion
clangStdenv.mkDerivation (finalAttrs: {
```